core/sawtooth/cli/transaction.py

```diff
@@ -155,26 +155,12 @@ def print_trans_info(args, web_client, tsctids):
         "Identifier": "TRANS",
         "InBlock": "BLOCK",
         "Status": "STATUS",
-        "TransactionType": "TXNTYPE",
-        "Update": {
```

Comment:
It's just my idea that an administrator has the role to register the permissioned validator. If we don't want to introduce an powerful node like an administrator. I'll change this file, and just get validators information like [an endpoint client](https://github.com/hyperledger/sawtooth-core/blob/master/core/sawtooth/endpoint_client.py).